actors.md

```diff
@@ -324,13 +324,35 @@ func SlashConsensusFault(block1, block2 BlockHeader) {
 		Fatal("blocks do not prove a slashable offense")
 	}
 
+	if !self.Miners.Has(msg.From) {
+		Fatal("slash collateral must only be called by a miner actor")
```

Comment:
Why?

Reply:
@decentralion My understanding was that you have to be a miner to have a collateral balance. Only those with a collateral balance (can be 0) can be rewarded when they call slash. I can totally be wrong on this.